modules/navier_stokes/src/userobjects/RhieChowInterpolatorBase.C

```diff
@@ -142,11 +140,12 @@ RhieChowInterpolatorBase::RhieChowInterpolatorBase(const InputParameters & param
     _velocity_interp_method = Moose::FV::InterpMethod::RhieChow;
 }
 
-bool
-RhieChowInterpolatorBase::hasFaceSide(const FaceInfo & fi, const bool fi_elem_side) const
+Real
```

Comment:
should you have an AD version of this?
I would be worried someone uses this inadvertently losing derivatives